site/templates/application/footer/nav.jet

```diff
@@ -9,5 +10,7 @@
         {{ end }}
       </ul>
     {{ end }}
+    
+    {{ yield socialMediaButtonsFooter() }}
```

Comment:
I'd like to stick with the existing naming convention (even though other people have already botched it a bit).  `footerSocialMediaButtons` would be better.

Reply:
Fixed this